lib/status/status_screen.dart

```diff
@@ -15,11 +15,13 @@
 * limitations under the License.
 */
 
-import 'dart:io';
+// dart:io not needed once thumbnails are rendered from memory
```

Comment:
remove dead code?

Reply:
normally I would greatly prefer if at least fully-dead code like this could be removed in the PR where it becomes dead

but for now i'll accept cleaning in follow-up, since I know the branch map is weird rn and i don't wanna make it worse